DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -13,7 +13,6 @@
     ViewData[LayoutViewDataKeys.HeaderPathName] ??= dlsSubApplication.HeaderPathName;
   }
   var headerExtension = (string)ViewData[LayoutViewDataKeys.Application];
-  var shouldDisplayHelpMenuItem = dlsSubApplication?.DisplayHelpMenuItem ?? headerExtension != DlsSubApplication.TrackingSystem.HeaderExtension;

```

Comment:
Do we need to tidy/remove the DisplayHelpMenuItem property from the dlsSubApplication class, I'm guessing it isn't used for anything else?

Reply:
It's been Removed